RecoLocalMuon/GEMRecHit/test/runGEMRecHitProducer_cfg.py

```diff
@@ -42,6 +42,14 @@
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```

Comment:
Hi Piet,

Thanks for the explanation.
The fix***Conditions are temporary workarounds for missing payloads.
I'd actually not put these hacks in the central files so that the "easy"
solutions are not propagated
and so that instead a more appropriate global tag is actually created
instead.
So, maybe put it in a separate file, like fixMissingUpgradeGTPayloads or
something.




On 4/13/15 10:37 AM, Piet Verwilligen wrote:
> In RecoLocalMuon/GEMRecHit/test/runGEMRecHitProducer_cfg.py
> <https://github.com/cms-sw/cmssw/pull/8524#discussion_r28249142>:
> 
>> @@ -42,6 +42,14 @@
>>  process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
> 
> Hi Slava
> 
> two things ...
> 1) since i was working on the factorization of the pull request I
> created two files within SimMuon/RPCDigitizer to test within the RPC
> Package whether everything works fine:
> 
>   * .../python/customizeRPCDigi.py (--> stolen from customizeGEMDigi.py
>     and adapted to have only RPC+DT+CSC only digitization. I find it
>     useful to have an easy way to have only muon digitization)
>   * .../test/runRPCDigiProducer_cfg.py
> 
> 2) i wrote the fixRPCConditions and fixDTConditions and fixCSCConditions
> to patch the upgrade global tags such that we can work on the upgrade of
> the muon system in 75X. Looking at
> Configurations/AlCa/python/autoCond.py the condition experts have put
> just placeholders. Once we have good conditions available most of it can
> be removed (note that for RPC it is not only loading different
> conditions, but also loading a different Digitization model) ... but it
> seems to be a pretty though road to get there ... So for now i prefer to
> have this solution.
> 
> But for those two points I will leave it up to you to decide where to
> put the code or how to deal with this kind of situations. I just
> committed how I have dealt with it. Maybe for maintenance it is better
> to have just one customize file and i put the one definition I added in
> customizeRPCDigi.py inside customizeGEMDigi.py?
> 
> Let me know.
> greets
> Piet
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8524/files#r28249142>.
> 

-- 
***********************************************************************
 Vyacheslav (Slava) Krutelyov
 TAMU: Physics Dept Texas A&M MS4242, College Station, TX 77843-4242
 CERN: 42-R-027
 AIM/Skype: siava16              googleTalk: slava77@gmail.com
 (630) 291-5128  Cell (US)       +41 76 275 7116 Cell (CERN)

-----------------------------------------------------------------------